### DISCUSSION TITLE
install with --package-lock-only flag should also update yarn.lock if present

### DISCUSSION THREAD

**Discussion by chris-convoy** [npm/feedback#188] [CLI] [answered]
Now that `npm install` [updates `yarn.lock` if present](https://github.blog/2021-02-02-npm-7-is-now-generally-available/#changes-to-the-lockfile), it would be amazing if the same behavior was applied when using the `--package-lock-only` flag. 

## Existing behavior:
- Run `npm install package@<version> --package-lock-only`
- `package-lock.json` is updated _without_ downloading `node_modules` 👍 👍  👍 
- `yarn.lock` is not kept in sync with `package-lock.json` 😢 

## Desired behavior:
- Run `npm install package@<version> --package-lock-only`
- `package-lock.json` is updated _without_ downloading `node_modules`  👍 👍  👍 
- `yarn.lock` is updated to stay in sync with the new package version, if present 🚀 

## Use case
See [this comment](https://github.com/yarnpkg/yarn/issues/5738#issuecomment-733231433) explaining how my team intends to use feature, and see [this comment](https://github.com/yarnpkg/yarn/issues/5738#issuecomment-770756001) about how the yarn team does not intend to bring feature parity between `yarn generate-lock-entry` and `npm install --package-lock-only`.

**Comment by MylesBorins** [comment, marked as answer]
@chris-convoy thanks for opening this discussion. Pinging @npm/cli to respond.

**Reply by MylesBorins** [reply]
@chris-convoy if this is something that you want to continue to explore a great next step would be to open an issue over at https://github.com/npm/rfcs/issues
